TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.html

```diff
@@ -14,15 +14,15 @@
 	</div>
 
 	<div class="neos-inspector-image-upload" id="{{unbound view._containerId}}">
-		{{view view.SecondaryInspectorButton viewClassBinding="view._mediaBrowserView" label="Media" action="_beforeMediaBrowserIsShown" target="view"}}
-		<button id="{{unbound view._browseButtonId}}" class="neos-button neos-inspector-image-upload-files" {{bindAttr disabled="view._uploadInProgress:neos-disabled"}}>
-			{{view._fileChooserLabel}}
+		{{view view.SecondaryInspectorButton viewClassBinding="view._mediaBrowserView" title="Media" icon="image" action="_beforeMediaBrowserIsShown" target="view"}}
+		<button id="{{unbound view._browseButtonId}}" class="neos-button neos-inspector-image-upload-files" {{bindAttr disabled="view._uploadInProgress:neos-disabled" title="view._fileChooserLabel"}}>
```

Comment:
we've previously talked about changing this label to "Upload" instead of "Choose", should we do that or is that unnecessary after this change?